### PR TITLE
Check for chainId before showing the UI

### LINKED ISSUE #3: Detect the network (eth_chainId?) and suggest to swtich to Rialot
Opening the page with Metamask isntalled will display ETH mainnet status, and might lead users to accidentally try to burn funds there and hope they are going to be relayed.

Instead the UI should check `chainId` and refuse to work if it doesn't match Rialto's chain Id (`0x69 = 105')

### LINKED ISSUE #9: Typo in "receiver" field
Says "Reciever" now.

### PR DESCRIPTION
**Pull request by Tbaut**
closes #9 
closes #3

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -37,15 +38,39 @@ const App = () => {
...
+		// Correctly handling chain changes can be complicated.
+		// We recommend reloading the page unless you have a very good reason not to.
+		window.location.reload();
+	});
+
+	if (Number(window.ethereum.networkVersion) !== EXPECTED_NETWORK_ID){
+		return (<ErrorMessage>
+			<>
+				<h1><Icon name='warning circle'/>Unexpected network</h1>
+				<h3>Please connect Metamask to Rialto (using {WS_PROVIDER})</h3>
```
Comment (tomusdrw) [inline src/App.tsx]: This should rather be `http://rpc.rialto.brucke.link:8545` not the `wss` stuff (which is Substrate node)

File: src/screens/EthToSub.tsx [hunk 2]
```diff
@@ -19,15 +19,15 @@ import useRialtoBlocks from '../hooks/useRialtoBlocks';
 import shortenAddress from '../util/shortenAddress';
 
 interface Props {
-    className?: string
-    ethProvider: ethers.providers.Web3Provider
+    className?: string;
+	ethProvider: ethers.providers.Web3Provider;
```
Comment (tomusdrw) [inline src/screens/EthToSub.tsx:23]: tabs?

**Review by tomusdrw — APPROVED** [review]
lgtm! Tiny address fix required.

File: src/App.tsx (hunk 1 above)
Comment (Tbaut) [reply src/App.tsx]: ah thanks

File: src/screens/EthToSub.tsx (hunk 2 above)
Comment (Tbaut) [reply src/screens/EthToSub.tsx:23]: yup, interesting that the linting didn't catch that.
edit: the line above was using white spaces, fixed it.

**Comment by Tbaut** [comment]
a bit of a catchall PR, worked the error message a bit.
error

**Review by tomusdrw — APPROVED** [review]

**Pull request merged**